pkg/auth/jwt.go

```diff
@@ -43,6 +44,9 @@ const (
 	OIDCServerURL = "OIDC_SERVER_URL"
 	// OpenidConfiguration is the discovery point on the OIDC server
 	OpenidConfiguration = ".well-known/openid-configuration"
+
+	// RSAPublicKey an RSA public key
+	RSAPublicKey = "RSA_PUBLIC_KEY"
```

Comment:
I don't see what this is for

Reply:
I should remove it. I don't know why it returned back in my branch.